net/wireguard-tools/files/patch-Makefile-sysconfdir-fix.diff

```diff
@@ -0,0 +1,27 @@
+--- src/tools/Makefile.orig	2018-07-18 12:20:48.000000000 -0700
++++ src/tools/Makefile	2018-07-26 16:37:32.000000000 -0700
+@@ -16,6 +16,9 @@
+ WITH_WGQUICK ?=
+ WITH_SYSTEMDUNITS ?=
+ 
++# cross-platform sed for in-place editing
++SED := $(shell sed --version >/dev/null 2>&1 && echo "sed -i --" || echo "sed -i ''")
++
+ ifeq ($(WITH_BASHCOMPLETION),)
+ ifneq ($(strip $(wildcard $(BASHCOMPDIR))),)
+ WITH_BASHCOMPLETION := yes
+@@ -79,10 +82,14 @@
+ 	install -v -d "$(DESTDIR)$(BASHCOMPDIR)" && install -v -m 0644 completion/wg.bash-completion "$(DESTDIR)$(BASHCOMPDIR)/wg"
+ 	@[ "$(WITH_WGQUICK)" = "yes" ] || exit 0; \
+ 	install -v -m 0755 wg-quick/$(PLATFORM).bash "$(DESTDIR)$(BINDIR)/wg-quick" && install -v -m 0700 -d "$(DESTDIR)$(SYSCONFDIR)/wireguard"
++	@[ "$(WITH_WGQUICK)" = "yes" -a "$(SYSCONFDIR)" != "/etc" ] || exit 0; \
++	$(SED) -e 's#^\(CONFIG_SEARCH_PATHS=.*\)\(/usr/local/etc\)#\1$(SYSCONFDIR)#' "$(DESTDIR)$(BINDIR)/wg-quick"
```

Comment:
Isn't `reinplace` in the Portfile or some similar tool more appropriate?

Reply:
I'll look into this.

My original intent with this patch was to have it eventually pushed upstream into the main source tree (e.g., this patch isn't MacPorts-specific). If using `reinplace` in the Portfile is preferred, I'll use that instead (since the replacement command should be the same).

Reply:
OK, I've replaced the patch with several `reinplace` calls in `post-destroot`.